webpack/assets/javascripts/services/ChartService.js

```diff
@@ -86,6 +93,42 @@ export const getDonutChartConfig = ({
     id,
   });
 
+export const getBarChartConfig = ({
+  data,
+  config,
+  onclick,
+  label,
+  id = uuidV1(),
+}) => {
+  const chartConfig = getChartConfig({
+    data,
+    config,
+    onclick,
+    id,
+  });
+
+  const categories = data.map(dataItem => dataItem[0]);
+
+  const columns = data.map(x => x[1]);
+
+  columns.unshift(label);
+
+  chartConfig.data.columns = [columns];
```

Comment:
I have mixed feeling about this. looks like we modify columns property in two different places,  if the data is calculated differently from donut chart, maybe it would be better to separate the logic between the two.

Reply:
I agree,  isn't this separation (between the two functions) enough?